### DISCUSSION TITLE
Repo organization

### DISCUSSION THREAD

**Discussion by nootrope** [lizlessner/Sensory-Engagement-Lab-2021#3] [General]
Should disparate patchers/projects be moved to their own directory/folder, with a respective README?

**Comment by lizlessner** [comment]
Thats a good idea thanks alberto. Any other organizational suggestions that
will help the repo stay navigable as it grows?

**Reply by nootrope** [reply]
Depending on how many subprojects are being tracked, it may become better to use Git Submodules. That way, each subproject would have its own Git repository linked to the overarching Sensory-Engagement-Lab-2021 repo. Don't think we're there yet.
